notes.txt

```diff
@@ -43,5 +43,7 @@ Details
 ---> Revenue
 ---> Runtime
 
+Videos?
```

Comment:
We briefly talked about this being it's own functional component or just being passed as props into the Details functional component.  I think if we want it's own section, we will need to make this a functional component.